dns/bind/src/opnsense/service/templates/OPNsense/Bind/blacklist.db

```diff
@@ -0,0 +1,12 @@
+$TTL    86400  
+@       IN      SOA     opnsense.local. hostmaster.opnsense.local. (
```

Comment:
you should never use `.local` as it is reserved for mdns (zeroconf). The hostname should use the data from the system settings.

Reply:
the hostmaster thing is btw. an email address and should be configurable.

Reply:
This is for static blacklist generation, for domain I'm open to anything, for email I wont make to to complicate to activate this feature since there will never be public request. 

When it comes to real zone configuration I'll add one field